src/assets/locales/es.json

```diff
@@ -1,169 +1,169 @@
   "media-browser.favorites-header": "Salas favoritas",
-  "media-browser.add_custom_object": "URL o archivo personalizado",
+  "media-browser.add_custom_object": "URL personalizado o archivo",
   "media-browser.add_custom_scene": "Escena personalizada",
-  "media-browser.add_custom_avatar": "URL GLB de Avatar",
+  "media-browser.add_custom_avatar": "URL del avatar en .GLB",
   "media-browser.privacy_policy": "Política de privacidad",
-  "media-browser.report_issue": "Informar problema",
-  "media-browser.powered_by.images": "Buscar por Bing | ",
-  "media-browser.powered_by.videos": "Buscar por Bing | ",
+  "media-browser.report_issue": "Reportar problema",
+  "media-browser.powered_by.images": "Búsqueda por Bing | ",
+  "media-browser.powered_by.videos": "Búsqueda por Bing | ",
   "media-browser.powered_by.youtube": "Búsqueda por Google | ",
-  "media-browser.powered_by.gifs": "Buscar por tenor |",
-  "media-browser.powered_by.sketchfab": "Buscar por Sketchfab | ",
+  "media-browser.powered_by.gifs": "Búsqueda por Tenor | ",
+  "media-browser.powered_by.sketchfab": "Búsqueda por Sketchfab | ",
   "media-browser.powered_by.poly": "Búsqueda por Google | ",
-  "media-browser.powered_by.twitch": "Buscar por Twitch | ",
-  "media-browser.powered_by.scenes": "Hecho con",
+  "media-browser.powered_by.twitch": "Búsqueda por Twitch | ",
+  "media-browser.powered_by.scenes": "Hecho con ",
   "media-browser.powered_by.avatars": " ",
-  "media-browser.empty.images": "No hay resultados. Intenta realizar una nueva búsqueda arriba.",
-  "media-browser.empty.videos": "No hay resultados. Intenta realizar una nueva búsqueda arriba.",
-  "media-browser.empty.youtube": "No hay resultados. Intenta realizar una nueva búsqueda arriba",
-  "media-browser.empty.gifs": "Ningún resultado. Intenta realizar una nueva búsqueda arriba.",
-  "media-browser.empty.sketchfab": "No hay resultados. Intenta realizar una nueva búsqueda arriba.",
-  "media-browser.empty.poly": "No hay resultados. Intenta realizar una nueva búsqueda arriba.",
-  "media-browser.empty.twitch": "No hay resultados. Intenta realizar una nueva búsqueda arriba.",
-  "media-browser.empty.favorites": "No tienes ningún favorito. Haz clic en ⭐ para agregarlo a tus favoritos",
+  "media-browser.empty.images": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.videos": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.youtube": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.gifs": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.sketchfab": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.poly": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.twitch": "No hay resultados. Prueba otra búsqueda arriba.",
+  "media-browser.empty.favorites": "No tienes ningún favorito. Haz clic en ⭐ para agregar a tus favoritos.",
```

Comment:
"No tienes ningún favorito. Haz clic en ⭐ para agregar a tus favoritos." should be "No tienes ningún favorito. Haz clic en ⭐ para agregarlo a tus favoritos."